src/test/java/org/hydev/mcpm/utils/HashUtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package org.hydev.mcpm.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+
+class HashUtilsTest
+{
+    HashUtils hashUtils = new HashUtils();
+
+    @Test
+    void hashString()
+    {
+        assert hashUtils.hash("hello world")
+            .equals("b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9");
+    }
+
+    @Test
+    void hashFile() throws IOException
+    {
+        assert hashUtils.hash(Objects.requireNonNull(GeneralUtils.getResourceFile("test-plugin-activelist.jar")))
```

Comment:
Regular `assert` again. Junit has some special handling for their Assertions class. For example, if you use `assertEquals` when a test fails you get the value of both the left hand and right hand side of the equals expression (neat!).

Reply:
fixed